src/Cassandra.IntegrationTests/Mapping/Tests/CreateTable.cs

```diff
@@ -72,14 +68,14 @@ public void CreateTable_MakeAllPropertiesCaseSensitiveAtOnce()
                 .TableName("tbl_case_sens_once")
                 .CaseSensitive());
 
-            var table = new Table<ManyDataTypesPoco>(_session, config);
+            var table = new Cassandra.Data.Linq.Table<ManyDataTypesPoco>(_session, config);
             table.Create();
 
             var mapper = new Mapper(_session, config);
-            ManyDataTypesPoco manyTypesInstance = ManyDataTypesPoco.GetRandomInstance();
+            var manyTypesInstance = ManyDataTypesPoco.GetRandomInstance();
             mapper.Insert(manyTypesInstance);
-            string cqlSelect = string.Format("SELECT * from \"{0}\" where \"{1}\"='{2}'", table.Name, "StringType", manyTypesInstance.StringType);
-            List<ManyDataTypesPoco> objectsRetrieved = mapper.Fetch<ManyDataTypesPoco>(cqlSelect).ToList();
+            var cqlSelect = $"SELECT * from \"{table.Name}\" where \"{"StringType"}\"='{manyTypesInstance.StringType}'";
```

Comment:
`\"{"StringType"}\"` within string interpolation can be simplified to the literal `\"StringType\"`.